reana_client/cli/workflow.py

```diff
@@ -712,11 +712,14 @@ def add_verbose_data_from_response(response, verbose_headers, headers, data):
 @click.option("--json", "json_format", count=True, help="Get output in JSON format.")
 @add_access_token_options
 @click.option(
-    "-s", "--step", "steps", multiple=True, help="Get logs of a specific step."
+    "--filter",
+    "filters",
+    multiple=True,
+    help="Filter the logs with key=value pair. Applicable filters are step, compute_backend, docker_img and status.",
```

Comment:
Please present available filters sorted everywhere in the help output: compute_backend, docker_img, status, step.

Reply:
Do you think "Available filters are.." is better than "Applicable filters are.."?

Reply:
Yeah, I think it better expresses that these filters are all *available*, and it is the user who at the end decides which ones will be *applied*.

(In the meantime I also suggested some wording for failure messages; see below.)